modules/nw-egress-ips-about.adoc

```diff
@@ -148,8 +148,15 @@ When the unreachable node comes back online, the egress IP address automatically
 
 [IMPORTANT]
 ====
-You cannot use manually assigned and automatically assigned egress IP addresses on the same nodes.
-If you manually assign egress IP addresses from an IP address range, you must not make that range available for automatic IP assignment.
+The following limitations apply when using egress IP addresses with the OpenShift SDN cluster network provider:
+
+- You cannot use manually assigned and automatically assigned egress IP addresses on the same nodes.
+- If you manually assign egress IP addresses from an IP address range, you must not make that range available for automatic IP assignment.
+- You cannot share egress IP addresses across multiple namespaces using the OpenShift SDN egress IP address implementation.
+ifeval::["{product-version}" >= "4.6"]
++
+If you need to share IP addresses across namespaces, the OVN-Kubernetes cluster network provider egress IP address implementation allows you to span IP addresses across multiple namespaces.
```

Comment:
There's nothing wrong with this, but the way it's formatted makes it look like there is a missing bullet in the output. Would it work to not force it onto the next line?

Reply:
@jeana-redhat, wasn't sure if that would work, but it does!